doc/Kokkos-Programming-Guide.md

```diff
@@ -0,0 +1 @@
+[Programming Guide](https://github.com/kokkos/ProgrammingGuide/wiki)
```

Comment:
Ups this should be kokkos/kokkos/wiki , ProgrammingGuide contains the private version.

Reply:
updated